src/Compilers/Core/Portable/Text/StringBuilderText.cs

```diff
@@ -21,7 +21,7 @@ internal sealed partial class StringBuilderText : SourceText
 
         private readonly Encoding? _encodingOpt;
 
-        public StringBuilderText(StringBuilder builder, Encoding? encodingOpt, SourceHashAlgorithm checksumAlgorithm)
+        public StringBuilderText(StringBuilder builder, Encoding? encodingOpt, SourceHashAlgorithm checksumAlgorithm = SourceHashAlgorithm.Sha1)
```

Comment:
Why this change?

Reply:
We needed to put Sha1 somewhere and this was consistent with the other APIs

Reply:
I'd prefer to keep the constructor as is and put the algorithm at the usage site. The only other usage of the constructor does explicitly need to pass along the algorithm from its own arguments, and for these scenarios I prefer to keep the knowledge where it's actually applicable.

Reply:
Done